purchases/build.gradle.kts

```diff
@@ -104,7 +104,13 @@ metalava {
 
     val name = if (variantName.lowercase().contains("defaults")) {
         excludeSourceSets.add("src/customEntitlementComputation/kotlin")
-        "api-defauts.txt"
+        if (variantName.lowercase().contains("bc8")) {
+            "api-defauts.txt"
```

Comment:
Noticed this typo before as well. If easy we can change it, if not let's leave it for another time :)

Reply:
I can't stop seeing it now